elyra/api/elyra.yaml

```diff
@@ -364,6 +409,84 @@ components:
           description: A free-form dictionary consisting of additional information
             about the resource.
 
+    PipelineResource:
+      description: The set of properties comprising a pipeline resource entity
+      required:
+        - primary_pipeline
+        - pipelines
+      type: object
+      properties:
+        primary_pipeline:
+          type: string
+          description: The primary pipeline id
+        pipelines:
+          type: array
+          description: A set of pipeline definitions
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The unique identifier of the pipeline
+              name:
+                type: string
+                description: The name of the pipeline
+              nodes:
+                type: array
+                description: The set of nodes in the pipeline
+                items:
+                  type: object
+                  properties: {}
+                  description: The node configuration
+              runtime:
+                type: string
+                description: The runtime type for the pipeline
+              runtime-config:
+                type: object
+                properties: {}
+                description: Runtime configuration that should be used to submit the pipeline
+
+    PipelineScheduleResponse:
+      description: The set of properties comprising a pipeline processor response entity
+      required:
+      - run-url
+      - object-storage-url
+      - object-storage-path
```

Comment:
Per your previous question regarding the use of '-' in response keys, I think we need to be consistent and I view '_' more appropriate (and common) for "code".  My vote would be that these should be changed to underscores.

Reply:
+1

Reply:
thanks. i agree and have updated the keys to be more consistent (i.e., use `_`)